sqlmesh/core/dialect.py

```diff
@@ -499,9 +499,14 @@ def parse(self: Parser) -> t.Optional[exp.Expression]:
             if isinstance(value, exp.Expression):
                 value.meta["sql"] = self._find_sql(start, self._prev)
 
-            expressions.append(self.expression(exp.Property, this=key, value=value))
-
-            if not self._match(TokenType.COMMA):
+            if self._match(TokenType.COMMA):
```

Comment:
I think we can just do `self._match(TokenType.COMMA, expression=prop)` here, assuming we did `prop = self.expression(...)` above this